src/formatDate.js

```diff
@@ -50,7 +50,44 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let str = date;
+
+  if (fromFormat[3] !== toFormat[3]) {
+    str = date.replace(fromFormat[3], toFormat[3]);
+    str = str.replace(fromFormat[3], toFormat[3]);
+  }
+
+  switch (fromFormat[0]) {
```

Comment:
you don't have default case and you're working only with first format from `fromFormat` variable. please use loop